Makefile.shared

```diff
@@ -96,6 +96,14 @@ XCODE_OPTIONS += CC=$(CC)
 endif
 endif
 
+ifeq ($(CODE_COVERAGE),YES)
+CONFIG_OPTIONS += --coverage
+else
+ifeq ($(CODE_COVERAGE),NO)
+CONFIG_OPTIONS += --no-coverage
```

Comment:
Isn't this the same as omitting CODE_COVERAGE?

Reply:
No, not if `set-webkit-configuration --coverage` was run prior to the `make [...] CODE_COVERAGE=NO` command.

Removing this would assume a pristine (clean) build directory before running the `make` command.